spec/controllers/jobs_controller_spec.rb

```diff
@@ -23,13 +23,41 @@ def self.expect_to_redirect_to_job_redirect_to
 
     context 'when the job is in progress' do
       it { is_expected.to respond_with(:accepted) }
+
+      context 'when the requested format is json' do
+        render_views
+        let(:format) { :json }
+        let(:json_response) { JSON.parse(response.body) }
+
+        it 'responses with an errored job status' do
+          expect(response.status).to be(200)
```

Comment:
Is there a difference between this line with `expect(response).to have_http_status(:ok)`?

I was thinking that it'd be nice if we can use Rails' "constants" for HTTP status codes.

Reply:
I dont think so but we are using this in general in our codebase so I am following the rest for consistency.